apps/frontend/src/app/PageTeam/CharacterDisplay/Build/Build.tsx

```diff
@@ -356,3 +353,93 @@ function BuildEditor({
     </CardThemed>
   )
 }
+
+type EquipBuildButtonProps = ButtonProps & { onEquip: () => void }
+function EquipBuildButton({
+  onEquip,
+  children,
+  ...props
+}: EquipBuildButtonProps) {
+  const [name, setName] = useState('')
+  const [copyEquipped, setCopyEquipped] = useState(false)
+  const [showPrompt, onShowPrompt, OnHidePrompt] = useBoolState()
+
+  const database = useDatabase()
+  const { teamCharId } = useContext(TeamCharacterContext)
+  const {
+    character: { equippedArtifacts, equippedWeapon },
+  } = useContext(CharacterContext)
+
+  const toEquip = () => {
+    if (copyEquipped) {
+      database.teamChars.newBuild(teamCharId, {
+        name: name !== '' ? name : 'Duplicate of Equipped',
+        artifactIds: equippedArtifacts,
+        weaponId: equippedWeapon,
+      })
+    }
+
+    onEquip()
+    setName('')
+    setCopyEquipped(false)
+    OnHidePrompt()
+  }
+  return (
+    <>
+      <Button {...props} onClick={onShowPrompt}>
+        {children}
+      </Button>
+      {/* TODO: Dialog Wanted to use a Dialog here, but was having some weird issues with closing out of it */}
+      {/* TODO: Translation */}
+      <ModalWrapper
+        open={showPrompt}
+        onClose={OnHidePrompt}
+        containerProps={{ maxWidth: 'md' }}
+      >
+        <CardThemed>
+          <CardHeader
+            title="Do you want to equip all gear in this build to this character?"
+            action={<CloseButton onClick={OnHidePrompt} />}
+          />
+          <CardContent sx={{ display: 'flex', flexDirection: 'column' }}>
+            <FormControlLabel
+              label="Copy my current equipment to a new loadout. Otherwise, the currently equipped build will be overwritten."
```

Comment:
```suggestion
              label="Copy my current equipment to a new build. Otherwise, the currently equipped build will be overwritten."
```

sorry, missed this

Reply:
no worries, pushed an update :)